### ISSUE TITLE
Should these panels work?

### ISSUE THREAD

**Issue by larson9999** [MVS-sysgen/sysgen#53]
I finally got MVS/CE working.  Looking the system over, I notice some panels do work.  Before I spend much time debugging I figured I'd ask if these panels should work.

1) =0.3 for setting pfkeys gives
------------------------  ISPF SERVICE ERROR DISPLAY   ------------------------
COMMAND ===>                                                                   
                                                           USERID   - IBMUSER 
 SERVICE NAME  DISPLAY                                     PANEL    - ISPERROR 
                                                           SCREEN   - 1       
 RETURN CODE   20                                          APPL ID  - ISP     
                                                           RELEASE  - V2.2.000 
                                                           SYSTEMID - MVSC    
                                                           LOGON    - IKJACCNT 
                                                                               
2) R  RAKF         RAKF Administrator Panels - Panel not found
3) G  Games        TSO Games  - Panel not found.

**Comment by mainframed** [comment]
Good catch, as of now they shouldn't go anywhere, I should probably remove those.

**Comment by larson9999** [comment]
> Good catch, as of now they shouldn't go anywhere, I should probably remove those.

Thanks for the reply.  Does this include the panels to view and set pfkeys?  That works for me in TK4 but not MVSCE.

**Comment by meltdown03** [comment]
The Foreground assembler (Option 4) doesn't work for me. I just did a complete sysgen today. I have other issues like IND$FILE crashing my logon job and forcing me to cancel the user

**Comment by doug4704** [comment]
how about option 4 (Foreground) for language processors. I'm on "unexpected sloth" 03/02/24. I get "PANEL NOT FOUND"
